modules/gdnative/gdnative_api.json

```diff
@@ -3122,6 +3122,14 @@
           ["godot_dictionary *", "p_self"]
         ]
       },
+      {
```

Comment:
That's almost correct. Once we made a release we must not change the old API entries. Instead a "core extension" has to be made for this. There already is the 1.1 extension, this would have to go into the 1.2 extension.